### DISCUSSION TITLE
VR Friendly, map on aircrafts cockpit screens

### DISCUSSION THREAD

**Discussion by psionic001** [albar965/littlenavmap#732] [Ideas]
Is there any way this Little NavMap can be placed on the glass cockpit screens of our Flight Sims?

I fly VR and have to keep peeking under the goggles to see how I'm profgressing on the map.

On MSFS2020 I fly the EC135 heli and it would be great to allow us to place a flightplan on one of the 3 screens with zoom in/out function.

Thanks

**Comment by KOKAProduktion** [comment]
Hiho @psionic001,

i had the same concern in VR and this solution worked for me: https://github.com/bymaximus/msfs2020-toolbar-little-nav-map
with the current fix: https://github.com/bymaximus/msfs2020-toolbar-little-nav-map/issues/9#issuecomment-798790183

Cheers!

**Comment by KOKAProduktion** [comment]
Related:
#747
#621
